### PR TITLE
Update Providing-Binary-Logs.md

### PR DESCRIPTION
**Pull request by jaredpar**
Need to include `""` around the path in the Powershell case

### CODE REVIEW

**Comment by rainersigwald** [comment]
Test failure is `ToolTaskThatTimeoutAndRetry` which should hopefully be addressed by #8659. Retrying.

**Review by rainersigwald — APPROVED** [review]

**Comment by jaredpar** [comment]
Do we need a second reviewer here? The Merge button is still grayed out

**Pull request merged**
